### ISSUE TITLE
(Feature request) Universal icon class.

### ISSUE THREAD

**Issue by mesasalata** [alandev27/intranet#1]
Add a rounded square cropped icon class for CSS. It should accept textures. Transparency in the textures should be rendered as transparent.
An example: 
<img width="304" alt="image" src="https://github.com/alandev27/intranet/assets/113854785/c58104da-748f-46aa-a25d-d9a938fe91d1">

It should have a modifiable size, the texture should be automatically cropped, and it should be tintable and
This class could also be used to make things like this:
<img width="82" alt="image" src="https://github.com/alandev27/intranet/assets/113854785/3db536c1-ece4-4c73-8a49-10823f247676">

Or for this:
<img width="148" alt="image" src="https://github.com/alandev27/intranet/assets/113854785/371b188c-0031-4611-a42d-7aa30570e24a">

Basically, it should be used as a universal icon class.
